packages/babel-plugin/src/utils/state-manager.js

```diff
@@ -146,6 +146,12 @@ export default class StateManager {
   }
 
   setOptions(options: { +[string]: mixed }): StyleXStateOptions {
+    const debug: StyleXStateOptions['debug'] = z.logAndDefault(
+      z.boolean(),
+      options.debug ?? false,
```

Comment:
Put this below dev and default it's value to `dev`.

```suggestion
      options.debug ?? dev,
```